great_expectations/self_check/util.py

```diff
@@ -1496,7 +1496,6 @@ def candidate_test_is_on_temporary_notimplemented_list_cfe(context, expectation_
             "expect_column_values_to_be_dateutil_parseable",
             "expect_column_values_to_be_json_parseable",
             "expect_column_values_to_match_json_schema",
-            "expect_column_stdev_to_be_between",
```

Comment:
Whoohoo. Just explicitly stating that this enables the `expect_column_stdev_to_be_between` Expectation tests for sql dialects (include `sqlite`). So additional unittests are not needed for this PR